### PR TITLE
Getting started markups

### PR DESCRIPTION
**Pull request by robbrockbank**
Updates and fixes to cloud config primarily for libnetwork

### CODE REVIEW

File: docs/getting-started/DigitalOcean.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ demo), and choose a region.  You should see something similar to the following:
 
 ![alt tag](digitalocean/Create_Droplet_1.png)
 
+You will creating two droplets, call the first calico-01 and the second
```
Comment (TrimBiggs) [inline docs/getting-started/DigitalOcean.md]: typo: "You will _be_ creating..."

File: docs/getting-started/DigitalOcean.md [hunk 2]
```diff
@@ -34,9 +36,18 @@ default docker networking.  Select the appropriate User Data based on the demons
...
-From the appropriate cloud-config directory, paste in the config `user-data-first` for the first host.
 
-And repeat this process for a second host, but this time paste in the config from `user-data-other`.
+For the first droplet `calico-01`, paste in the cloud config from
+`user-data-first`.
+
+When the first droplet is running, look at the settings to get its private IPv4
+address.
+
+And repeat this process for a second host `calico-02`, but this time use the
```
Comment (TrimBiggs) [inline docs/getting-started/DigitalOcean.md]: Unnecessary "And "

**Comment by tomdee** [comment]
Squash please

**Comment by tomdee** [comment]
Looks good

**Pull request merged**